server/graphql_schemas/schema.py

```diff
@@ -27,6 +27,7 @@ class Mutation(
     AttendMutation,
     CategoryMutation,
     ImageMutation,
+    AndelaUserMutation,
```

Comment:
This should follow the alphabetical order. 
```
AndelaUserMutation,
AttendMutation,
CategoryMutation,
ImageMutation,
```
this should be applicable to other imports.